### PR TITLE
Backend - Game Class and API

### PR DESCRIPTION
**Pull request by awb8593**
The Pi will now take all of the games located in the Pi's game directory and store them in a dictionary accessible by the frontend. The games are stored as Game objects which have all of their necessary attributes.

### CODE REVIEW

**Comment by zmb6893** [comment]
@awb8593  Have you been able to confirm this works with a game already on the pi like minecraft???

**Comment by awb8593** [comment]
@zmb6893 The new attributes have not been tested on the pi, only on my local machine, but the everything before the final 2 commits did get tested on it (launching it, getting the name and file path), so the changes that haven't been tested on it are the newly added attributes

**Comment by zmb6893** [comment]
Alright Mr. Burgos (@awb8593), it's your turn for code review now, part 2 👀

File: src/backend/game_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from fastapi import FastAPI
+import game
+import launcher
+import json
+
+app = FastAPI()
+file_path = r"C:\Users\awbus\OneDrive\Desktop\games.json"
```
[the remark this answers is not in the document]
Comment (awb8593) [reply src/backend/game_endpoints.py:7]: good catch, this should actually be the path on the pi so I will change it to that

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```
[the remark this answers is not in the document]
Comment (awb8593) [reply .vscode/launch.json:3]: sure thing

File: src/backend/__pycache__/game.cpython-310.pyc
[the remark this answers is not in the document]
Comment (awb8593) [reply src/backend/__pycache__/game.cpython-310.pyc:1]: on it

**Comment by zmb6893** [comment]
@awb8593 Looks good to me! The only think I would note is that in the json file, there is still your computer's file path, but we can change this in the future when we load the games up onto the console!

Feel free to merge this!

**Comment by awb8593** [comment]
@zmb6893 Ah, games.json should be blank since its based on the pi's local directory. Committing a blank one now before merging

**Pull request merged**